packages/kit/src/core/create_manifest_data/index.js

```diff
@@ -238,9 +239,38 @@ export default function create_manifest_data({ config, output, cwd = process.cwd
 	walk(config.kit.files.routes, [], [], [layout], [error]);
 
 	const assets_dir = config.kit.files.assets;
+	/**
+	 * @type {import('types/internal').Asset[]}
+	 */
+	let assets = [];
+	if (fs.existsSync(assets_dir)) {
+		/**
+		 * @type {string[]}
+		 */
+		const exclusions = config.kit.serviceWorker.exclude || [];
+
+		// .DS_STORE files are automatically removed to keep the compatiblity
```

Comment:
compatibility is spelled wrong. I think we can just remove this line though
```suggestion
```

Reply:
Done. Thanks for pointing it out.
Added Code Spell check extension as well.